File: Day1/Python3/fizzbuzz2.py
'''
 * @author: ashwek
 * @date: 20/12/2018
'''

n = int(input("Enter limit : "))

for i in range(1, n+1):
    
    output = ""
    if i%3 == 0 :
        output = "Fizz"
    if i%5 == 0:
        output += "Buzz"

    if output :
        print(output)
    else:
        print(i)

File: Day1/C/fizzbuzz2.c
/*
        * @author: shashanknenar
        * @date: 01/10/2020
*/


#include <stdio.h> 
  
int main(void) 
{ 
    int i; 
    for (i=1; i<=100; i++) 
    { 

        if (i%15 == 0)         
            printf ("FizzBuzz\t");     

        else if ((i%3) == 0)     
            printf("Fizz\t");                  

        else if ((i%5) == 0)                        
            printf("Buzz\t");                  
      
        else            
            printf("%d\t", i); //print                 
    } 
  
    return 0; 
} 